src/sass/Fabric.Brand.Icons.Output.scss

```diff
@@ -0,0 +1,82 @@
+// Images Path
+$imagesPath: "https://appsforoffice.microsoft.com/fabric/assets/Product/PNG";
+
+// Icon Names
+$iconList: Access Excel InfoPath Office OneDrive OneNote Outlook PowerPoint Project SharePoint Visio Word;
+
+// Icon Sizes
+$iconSizes: 16 48 96;
+
+// Mixin to loop through iconList and generate classes
+@mixin iconClass($size, $retina) {
+  @each $name in $iconList {
+    &.ms-Brand--#{$name}#{$size} {
+        @include Icon($name, $retina, $size);
+    }
+  }
+}
+
+// Generate icon size classes
+@mixin IconSizeClass() {
```

Comment:
Let's put the mixins and variables in `_Fabric.Brand.Icons.scss` so that these can be used in other apps without outputting any CSS.